modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -213,6 +213,85 @@ public void restClientClusterHealth() throws IOException {
         // }
     }
 
+    @Test
+    public void restClientClusterHealthElasticsearch8() throws IOException {
+        // httpClientContainer8 {
+        // Create the elasticsearch container.
+        try (
+            ElasticsearchContainer container = new ElasticsearchContainer(
+                "docker.elastic.co/elasticsearch/elasticsearch:8.1.2"
+            )
+        ) {
+            // Start the container. This step might take some time...
+            container.start();
+
+            // Do whatever you want with the rest client ...
+            final CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
+            credentialsProvider.setCredentials(
+                AuthScope.ANY,
+                new UsernamePasswordCredentials(ELASTICSEARCH_USERNAME, ELASTICSEARCH_PASSWORD)
+            );
+
+            client =
+                RestClient
+                    // use HTTPS for Elasticsearch 8
+                    .builder(HttpHost.create("https://" + container.getHttpHostAddress()))
+                    .setHttpClientConfigCallback(httpClientBuilder -> {
+                        httpClientBuilder.setDefaultCredentialsProvider(credentialsProvider);
+                        // SSL is activated by default in Elasticsearch 8
+                        httpClientBuilder.setSSLContext(container.createSslContextFromCa());
+                        return httpClientBuilder;
+                    })
+                    .build();
+
+            Response response = client.performRequest(new Request("GET", "/_cluster/health"));
+            // }}
+            assertThat(response.getStatusLine().getStatusCode()).isEqualTo(200);
+            assertThat(EntityUtils.toString(response.getEntity())).contains("cluster_name");
+            // httpClientContainer8 {{
```

Comment:
```suggestion
            // httpClientContainer8 {
```

Reply:
In line 211 it is also `{{`. Which one is correct?

Reply:
It should be a single curly brace.

Reply:
I tested it and it only works with a single curly brace here. Docs preview looks good to me now